### DISCUSSION TITLE
SMBX 1.3 Config Pack missing a lot objects

### DISCUSSION THREAD

**Discussion by agergo100** [WohlSoft/PGE-Project#444] [Q&A]
![Screenshot_20210520_181728_ru wohlnet pge editor](https://user-images.githubusercontent.com/51967530/119015064-daaefa00-b998-11eb-9fca-d8d31a82842c.jpg)

I just followed the instructions.
Download the Config Pack from this page
http://wohlsoft.ru/config_packs/
and the PGE show this.

**Comment by Wohlstand** [comment]
The Android version of Editor (0.3.0.3) is too old to use modern config packs, and it requires the compatible config pack:
Here is the preserved config pack that will be compatible with that Editor version:
http://wohlsoft.ru/PGE/_downloads/legacy/SMBX-2016-config-pack.7z

**Comment by agergo100** [comment]
Everything is fine but do you thinking about updating the PGE android?
![Screenshot_20210520_191302_ru wohlnet pge editor](https://user-images.githubusercontent.com/51967530/119021240-7c394a00-b99f-11eb-87bf-2d8fd85c07c0.jpg)

**Comment by Wohlstand** [comment]
Keep the note: "PGE" is the codename of entire project that includes many components, the Editor is only a small part of it

> Everything is fine but do you thinking about updating the PGE android?

This build you do have is a result of an unsuccessful experiment on attempting to run the Editor on a mobile device in as-is form. There are some plans on making the different sort of Editor for mobiles that will have a different UI targeted to be used on a touch screen having to utilize multitouch and gestures for convenient use. This one just sucks, because you can't properly use it on a touch screen without tricks, and no audio working to play music.

**Comment by agergo100** [comment]
I see

**Comment by Wohlstand** [comment]
Okay, I close this as this question is solved.
